hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright The Kubernetes Authors.
```

Comment:
```suggestion
Copyright The Kangal Authors.
```

Reply:
Let's do this in a separate PR that also updates all the code-generated files